db/migrate/20171107235209_create_pieces.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePieces < ActiveRecord::Migration[5.0]
+  def change
+    create_table :pieces do |t|
+      t.string :color
+      t.string :type
+      t.integer :x_position
+      t.integer :y_position
+      t.timestamps
+    end
```

Comment:
Needs `game_id`

And `belongs_to :game` in the model.

Reply:
We will add the `game id` in piece migration asap. We already have `belongs_to :game` in the piece model.